special-pages/pages/new-tab/app/favorites/components/FavoritesCustomized.js

```diff
@@ -0,0 +1,54 @@
+import { h } from 'preact';
+import { useContext } from 'preact/hooks';
+
+import { useTypedTranslation } from '../../types.js';
+import { useVisibility } from '../../widget-list/widget-config.provider.js';
+import { useCustomizer } from '../../customizer/Customizer.js';
+
+import { FavoritesContext, FavoritesProvider } from './FavoritesProvider.js';
+import { PragmaticDND } from './PragmaticDND.js';
+import { FavoritesMemo } from './Favorites.js';
+
+/**
+ * Component that consumes FavoritesContext for displaying favorites list.
+ */
+export function FavoritesConsumer() {
+    const { state, toggle, favoritesDidReOrder, openContextMenu, openFavorite, add } = useContext(FavoritesContext);
+
+    if (state.status === 'ready') {
+        return (
+            <PragmaticDND items={state.data.favorites} itemsDidReOrder={favoritesDidReOrder}>
+                <FavoritesMemo
+                    favorites={state.data.favorites}
+                    expansion={state.config.expansion}
+                    openContextMenu={openContextMenu}
+                    openFavorite={openFavorite}
+                    add={add}
+                    toggle={toggle}
+                />
+            </PragmaticDND>
```

Comment:
This helps to de-couple the D+D implementation from the favorites grid - mostly to help us switch libraries or turn it off easily in the future